src/components/menu/__internal__/submenu/submenu.style.ts

```diff
@@ -23,6 +24,7 @@ interface StyledSubmenuProps
     Pick<SubmenuProps, "variant"> {
   submenuDirection?: string;
   maxHeight?: string;
+  applyFocusRadiusStyling?: boolean;
```

Comment:
```suggestion
  applyFocusRadiusStyling: boolean;
```

If we always coerce the value to a bool it doesn't need to be optional here